static/src/javascripts/projects/commercial/modules/messenger/click.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import { trackNativeAdLinkClick } from 'common/modules/analytics/google';
+import { register } from 'commercial/modules/messenger';
+
+const sendClick = (adSlot: Element, linkName: string): void => {
+    trackNativeAdLinkClick(adSlot.id, linkName);
+};
+
+register('click', (linkName, ret, iframe = {}) =>
+    sendClick(
+        iframe.closest('.js-ad-slot') || {
+            id: 'unknown',
+        },
+        linkName || ''
+    )
+);
+
+export { sendClick };
```

Comment:
Do you even need to export it? It doesn't look like the function is being imported anywhere (sorry if this short-circuits the dilemma)